File: training-center-client/src/app/Model/instructor.ts
import { Course } from "./Course";


export interface instructor {

    id : number;
    Name : string;
    Email : string
    Specialization : string;
    courses : Course[];
}


export interface CreateInstructor {
    Name : string;
    Email : string;
    Specialization : string;

}

export interface UpdateInstructor {
    Name : string;
    Email : string;
    Specialization : string;
}



export interface InstructorResponse{
    id : number;
    Name : string;
    Email : string
    Specialization : string;
}

File: TrainingCenter.Api/Model/Instructor.cs
using System.ComponentModel.DataAnnotations;
namespace TRAININGCENTER.API.Model;
public class Instructor
{
    
    public int id {get; set;}

    public string Name {get;set;} = string.Empty;

    public string Email {get;set;} = string.Empty;

    public string Specialization {get; set;} = string.Empty;

    public List<Course> courses {get;set;}  = new();

}